README.md

```diff
@@ -117,9 +240,12 @@ the param value must be set to one of the "true" values above, e.g. `?active=tru
 
 ## Block usage
 
-`has_scope` also accepts a block. The controller, current scope and value are yielded
-to the block so the user can apply the scope on its own. This is useful in case we
-need to manipulate the given value:
+`has_scope` also accepts a block in case we need to manipulate the given value. Usually there are three arguments passed down in the block:
```

Comment:
```suggestion
`has_scope` also accepts a block in case we need to manipulate the given value and/or call the scope in some custom way. Usually three arguments are passed to the block:
```